### ISSUE TITLE
I get "error: '/dev/video0' is not a valid webcam input device on this system" when trying to record video w/o webcam

### ISSUE THREAD

**Issue by Hund** [dbermond/screencast#2]
Not sure if I'm doing something wrong?

```
./screencast -i default -f mp4 -S -u
[ screencast ] error: '/dev/video0' is not a valid webcam input device on this system
```

I'm using version 1.4.0 on Funtoo anyway.

**Comment by dbermond** [comment]
Thank you for the interest in screencast.

I have identified the problem and will fix it shortly.

Offtopic hint:
You don't need to specify `-f mp4` because this is the default. You can simply omit this option and the output format will be mp4. The same logic applies to all options.

**Comment by Hund** [comment]
Cool, thanks! :) Thank you yourself, for making this awesome tool! It's a lot more easier to use this, than just ffmpeg alone.

**Comment by dbermond** [comment]
Fixed by commit 63593e9c5ac0ddb658c7ae6c9d3e646dd580de00.
Auto closed by GitHub.

> Cool, thanks! :) Thank you yourself, for making this awesome tool! It's a lot more easier to use this, than just ffmpeg alone.

It's good to know that this program is being useful for someone. Thank you for using it.
I'm planning a gui version for the future.
